### ISSUE TITLE
1.5.0: call for testing

### ISSUE THREAD

**Issue by martelletto** [Yubico/libfido2#193]
Hi,

A new version of libfido2 will likely be released in September. If you would like to ensure libfido2 works with your key/platform/application, now would be a good time to test git HEAD and report any concerns.

Thank you,

-p.

**Comment by sb230132** [comment]
hi,
Iam testing 1.5.0 and see an issue:
I have raised the issue in following link: https://github.com/Yubico/libfido2/issues/195

**Comment by sb230132** [comment]
I want to understand what is the below code doing ??
	do {
		if (fido_tx(dev, CTAP_CMD_MSG, iso7816_ptr(apdu),
		    iso7816_len(apdu)) < 0) {
			fido_log_debug("%s: fido_tx", __func__);
			r = FIDO_ERR_TX;
			goto fail;
		}
		if ((reply_len = fido_rx(dev, CTAP_CMD_MSG, &reply,
		    sizeof(reply), ms)) < 2) {
			fido_log_debug("%s: fido_rx", __func__);
			r = FIDO_ERR_RX;
			goto fail;
		}
		if (usleep((unsigned)(ms == -1 ? 100 : ms) * 1000) < 0) {
			fido_log_debug("%s: usleep", __func__);
			r = FIDO_ERR_RX;
			goto fail;
		}
	} while (((reply[0] << 8) | reply[1]) == SW_CONDITIONS_NOT_SATISFIED);

In short iam guessing that fido_tx is sending client data hash and RP id to device.
fido_rx is reading the response from the device.

**Comment by martelletto** [comment]
From the code excerpt above, all I can tell is that an APDU is transmitted to the key and a reply read until the key replies with a status word different than `SW_CONDITIONS_NOT_SATISFIED`. For more context, please link to the particular code in u2f.c.

**Comment by martelletto** [comment]
It has been 36 days; closing. Thank you for your interest in testing libfido2, @sb230132.
